src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ExportDefaultProjectEngineFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+#nullable disable
+
+using Microsoft.AspNetCore.Razor.ProjectEngineHost;
+
+namespace Microsoft.CodeAnalysis.Razor.Workspaces;
+
+[ExportCustomProjectEngineFactory("Default", SupportsSerialization = true)]
+internal class ExportDefaultProjectEngineFactory : DefaultProjectEngineFactory
```

Comment:
Another unnecessary link between functionality and MEF. `ProjectEngineFactory` was moved to ProjectEngineHost, and this now does the exporting.